pkg/kubelet/certificate/certificate_manager_test.go

```diff
@@ -527,6 +533,42 @@ func TestInitializeCertificateSigningRequestClient(t *testing.T) {
 	}
 }
 
+func TestNewKubeletClientCertificateManager(t *testing.T) {
```

Comment:
Since this test uses `NewKubeletClientCertificateManager` it should be in `kubelet_test.go`. There is a [PR](https://github.com/kubernetes/kubernetes/pull/49654) to move `certificate_manager` and this test file to another location where they won't be colocated with the `kubelet.go` file.